blaze-client/src/main/scala/org/http4s/client/blaze/package.scala

```diff
@@ -7,6 +7,7 @@ package object blaze {
 
   /** Default blaze client
     *
-    * This client will create a new connection for every request. */
-  lazy val defaultClient: Client = SimpleHttp1Client(BlazeClientConfig.defaultConfig())
+    * This client will create a new connection for every request.
+    */
+  lazy val defaultClient: Client = SimpleHttp1Client(BlazeClientConfig.defaultConfig)
```

Comment:
Agreed. In fact, I had deprecated it but decided that it should be a separate PR so I reverted that change. :)